man/nvim.1

```diff
@@ -336,8 +336,22 @@ Print version information and exit.
 .El
 .Sh ENVIRONMENT
 .Bl -tag -width Fl
+.It Ev XDG_CONFIG_HOME
+Path to use for the user-local runtime directory, see
+.Sx FILES .
+Defaults to
+.Pa ~/.config
```

Comment:
Does Windows even have man pages? If not then I'm guessing it doesn't matter here.